lib/offsite_payments/integrations/realex.rb

```diff
@@ -3,7 +3,8 @@ module Integrations #:nodoc:
     module Realex
       mattr_accessor :production_url
       mattr_accessor :test_url
-      self.production_url = 'https://hpp.realexpayments.com/pay'
+      # self.production_url = 'https://hpp.realexpayments.com/pay'
```

Comment:
Okay so lets just leave it with https://epage.payandshop.com/epage.cgi as the sole URL for the moment.